README.rst

```diff
@@ -1,97 +1,118 @@
-==========
-Tahoe-LAFS
-==========
+======================================
+Free and Open decentralized data store
+======================================
 
-Tahoe-LAFS is a Free and Open decentralized cloud storage system. It
-distributes your data across multiple servers. Even if some of the servers
-fail or are taken over by an attacker, the entire file store continues to
-function correctly, preserving your privacy and security.
+|image0|
 
-For full documentation, please see
-http://tahoe-lafs.readthedocs.io/en/latest/ .
+`Tahoe-LAFS <https://www.tahoe-lafs.org>`__ (Tahoe Least-Authority File Store) is the first free software / open-source storage technology that distributes your data across multiple servers. Even if some servers fail or are taken over by an attacker, the entire file store continues to function correctly, preserving your privacy and security.
 
 |Contributor Covenant|  |readthedocs|  |travis|  |circleci|  |codecov|
 
 
-INSTALLING
-==========
+Table of contents
 
-There are three ways to install Tahoe-LAFS.
+-  `About Tahoe-LAFS <#about-tahoe-lafs>`__
 
-using OS packages
-^^^^^^^^^^^^^^^^^
+-  `Installation <#installation>`__
 
-Pre-packaged versions are available for several operating systems:
+-  `Contributing <#contributing>`__
```

Comment:
The index doesn't match the new section order.  In the index the section order is "Installation", "Contributing" but in the body the order is "Installation", "Community", "Contributing".